en/renew-tls-certificate.md

```diff
@@ -4,3 +4,144 @@ summary: Learn how to renew and replace TLS certificates between TiDB components
+    kubectl create secret generic ${tikv_group_name}-tikv-cluster-secret --namespace=${namespace} --from-file=tls.crt=tikv-server.pem --from-file=tls.key=tikv-server-key.pem --from-file=ca.crt=ca.pem --dry-run=client -o yaml | kubectl apply -f -
+    kubectl create secret generic ${tidb_group_name}-tidb-cluster-secret --namespace=${namespace} --from-file=tls.crt=tidb-server.pem --from-file=tls.key=tidb-server-key.pem --from-file=ca.crt=ca.pem --dry-run=client -o yaml | kubectl apply -f -
+    ```
+
+    In the preceding command, `${pd_group_name}`, `${tikv_group_name}`, and `${tidb_group_name}` are the names of the component groups, and `${namespace}` is the namespace in which the TiDB cluster is deployed.
+
+    > **Note:**
+    >
+    > The preceding command only renews the server-side and the client-side certificate between PD, TiKV, and TiDB components. If you need to renew the server-side certificates for other components, such as TiCDC, TiFlash and TiProxy, you can execute the similar command.
+
+3. [Perform the rolling restart](restart-a-tidb-cluster.md) to components that need to load the new certificates.
+
+    After the completion of the rolling restart, each component uses the new certificate for TLS communication. If you refer to [Renew and replace the CA certificate](#renew-and-replace-the-ca-certificate) and make each component load the combined CA certificate, each component can still accept the certificate issued by the original CA certificate.
+
+### Optional: Remove the original CA certificate from the combined CA certificate
+
+After you [renew and replace the combined CA certificate](#renew-and-replace-the-ca-certificate), [server-side and client-side certificates between components](#renew-and-replace-certificates-between-components), you might want to remove the original CA certificate (for example, because the CA certificate has expired or the private key is compromised). To remove the original CA certificate, take steps as follows:
+
+1. Renew the Kubernetes Secret objects based on the new CA certificate.
+
+    ```shell
+    kubectl create secret generic ${cluster_name}-pd-cluster-secret --namespace=${namespace} --from-file=tls.crt=pd-server.pem --from-file=tls.key=pd-server-key.pem --from-file=ca.crt=ca.new.pem --dry-run=client -o yaml | kubectl apply -f -
+    kubectl create secret generic ${cluster_name}-tikv-cluster-secret --namespace=${namespace} --from-file=tls.crt=tikv-server.pem --from-file=tls.key=tikv-server-key.pem --from-file=ca.crt=ca.new.pem --dry-run=client -o yaml | kubectl apply -f -
+    kubectl create secret generic ${cluster_name}-tidb-cluster-secret --namespace=${namespace} --from-file=tls.crt=tidb-server.pem --from-file=tls.key=tidb-server-key.pem --from-file=ca.crt=ca.new.pem --dry-run=client -o yaml | kubectl apply -f -
+    ```
+
+    In the preceding command, `${cluster_name}` is the name of the cluster, and `${namespace}` is the namespace in which the TiDB cluster is deployed.
```

Comment:
Inconsistent variable for secret names, should be `${pd_group_name}-pd-cluster-secret`?

Reply:
https://github.com/pingcap/docs-tidb-operator/pull/2824/commits/fbd2c9adc975ae263efd379fe3898057283d6a4a